.github/workflows/tests.yaml

```diff
@@ -28,11 +28,10 @@ jobs:
       uses: actions/checkout@v4
     - uses: actions/setup-node@v4
       with:
-        node-version: '16'
+        node-version: '20'
```

Comment:
Can we test with node 22 ? (even though we enforce node 20 in the package.json) should be great to test with latest version